src/Search.js

```diff
@@ -0,0 +1,73 @@
+import React, { Component } from "react";
+import { ReactSearchAutocomplete } from "react-search-autocomplete";
+import "./search.css";
+
+const autocompleteURL =
+  "https://autocomplete.search.hereapi.com/v1/autocomplete?";
+
+function autoCompleteCity(city) {
+  if (!city) return Promise.resolve("");
```

Comment:
Why are you returning empty string when city parameter is undefined. Why not an empty array?

Reply:
resolved